easybuild/easyconfigs/l/Leafcutter/leafcutter-aa12b1e-foss-2018b.eb

```diff
@@ -0,0 +1,37 @@
+easyblock = 'PackedBinary'
+
+name = 'leafcutter'
+version = 'aa12b1e'
```

Comment:
A date is missing in the name of the eb file. Version should be something like: YYYYMMDD-aa12b1e